index.html

```diff
@@ -0,0 +1,106 @@
+<!DOCTYPE html>
+<html lang="en" dir="ltr">
+  <head>
+    <meta charset="utf-8">
+    <title>Tribute Page</title>
+  <link rel="stylesheet" href="style.css">
```

Comment:
Please include the meta view port as stated in the linter errors you can check errors occurred by running "npx hint ."